package.json

```diff
@@ -31,31 +31,31 @@
   },
   "homepage": "https://github.com/yext/answers-hitchhiker-theme#readme",
   "devDependencies": {
-    "@axe-core/puppeteer": "^4.3.1",
+    "@axe-core/puppeteer": "^4.5.2",
     "@babel/core": "^7.9.6",
     "@babel/preset-env": "^7.9.6",
     "@percy/cli": "^1.0.0-beta.67",
-    "@percy/puppeteer": "^2.0.0",
+    "@percy/puppeteer": "^2.0.2",
     "@types/jest": "^26.0.19",
     "@yext/cta-formatter": "^1.0.0",
     "babel-jest": "^25.5.1",
     "comment-json": "^4.1.1",
     "cross-env": "^7.0.2",
     "currency-symbol-map": "^5.0.1",
     "escape-html": "^1.0.3",
-    "express": "^4.17.1",
+    "express": "^4.18.2",
     "file-system": "^2.2.2",
     "full-icu": "^1.3.1",
     "handlebars": "^4.7.7",
     "jambo": "^1.12.3",
     "jest": "^25.5.2",
     "libphonenumber-js": "^1.9.6",
-    "loader-utils": "^2.0.0",
+    "loader-utils": "^2.0.4",
     "locale-currency": "0.0.2",
     "lodash.clonedeep": "^4.5.0",
     "postcss": "^8.4.14",
-    "puppeteer": "^10.2.0",
-    "serve": "^11.3.2",
+    "puppeteer": "^13.7.0",
+    "serve": "^14.1.2",
```

Comment:
I think this `serve` version drops support of node 12. I assume we're fine with that since we've stopped testing on node 12? and if so, do we need to update the package.json's required node engine to reflect the change?

Reply:
That is a good idea. If it's agreed we should only support versions we test on then I can set the node version target to >14. If we're not sure I'll downgrade `serve`, to a version that has the same node support as originally.

Reply:
I'm seeing @percy/puppeteer supports [>=14](https://github.com/percy/percy-puppeteer/blob/master/package.json#L20) for v2.0.2

Reply:
hmm reverting this didn't work because the change was just a patch. I forced it

Reply:
I still think it would be worth checking if Node 12 support is no longer necessary. It would be nice to get these fixes if we can